lib/swagger-client.js

```diff
@@ -39,10 +39,19 @@ class Root extends Component {
    */
   loadSpec() {
     return new Promise((resolve, reject) => {
-      this.http.request('GET', { path: '/swagger.json' }, (err, res) => {
+      this.http.request('GET', { headers: { accept: 'application/json' }, path: '/openapi/v2' }, (err, res) => {
```

Comment:
Is the `headers` required?  I would expect https://github.com/godaddy/kubernetes-client/blob/master/lib/request.js#L147 to do that implicitly.

Reply:
You're right, it's unnecessary, I explicitly added it because the k8s docs (https://kubernetes.io/docs/concepts/overview/kubernetes-api/#openapi-and-swagger-definitions) use headers to differentiate between json and pb swaggers, but k8s also explicitly uses `application/json` if header is not set, will remove.